fbw-a32nx/src/systems/shared/src/NavigationDisplay.ts

```diff
@@ -36,22 +36,28 @@ export enum NdSymbolTypeFlags {
     ActiveLegTermination = 1 << 7,
     EfisOption = 1 << 8,
     Dme = 1 << 9,
-    ConstraintMet = 1 << 10,
-    ConstraintMissed = 1 << 11,
-    ConstraintUnknown = 1 << 12,
-    SpeedChange = 1 << 13,
-    FixInfo = 1 << 14,
-    FlightPlan = 1 << 15,
-    PwpDecel = 1 << 16,
-    PwpTopOfDescent = 1 << 17,
-    PwpCdaFlap1 = 1 << 18,
-    PwpCdaFlap2 = 1 << 19,
-    FlightPlanVectorLine = 1 << 20,
-    FlightPlanVectorArc = 1 << 21,
-    FlightPlanVectorDebugPoint = 1 << 22,
-    ActiveFlightPlanVector = 1 << 23,
-    CourseReversalLeft = 1 << 24,
-    CourseReversalRight = 1 << 25,
+    Constraint = 1 << 10,
+    FixInfo = 1 << 11,
+    FlightPlan = 1 << 12,
+    FlightPlanVectorLine = 1 << 13,
+    FlightPlanVectorArc = 1 << 14,
+    FlightPlanVectorDebugPoint = 1 << 15,
+    ActiveFlightPlanVector = 1 << 16,
```

Comment:
These are not needed AFAIK.

Reply:
Yeah, good idea. @Benjozork Do you think we can do this or will it make the merge harder?